JitsiConference.js

```diff
@@ -595,13 +595,21 @@ JitsiConference.prototype.addTrack = function(track) {
 };
 
 /**
- * Fires TRACK_AUDIO_LEVEL_CHANGED change conference event.
+ * Fires TRACK_AUDIO_LEVEL_CHANGED change conference event (for local tracks).
+ * @param {TraceablePeerConnection|null} tpc
  * @param audioLevel the audio level
  */
-JitsiConference.prototype._fireAudioLevelChangeEvent = function(audioLevel) {
-    this.eventEmitter.emit(
-        JitsiConferenceEvents.TRACK_AUDIO_LEVEL_CHANGED,
-        this.myUserId(), audioLevel);
+JitsiConference.prototype._fireAudioLevelChangeEvent
+= function(tpc, audioLevel) {
+    const activeTpc = this.getActivePeerConnection();
+
+    // There can be no TraceablePeerConnection if audio levels do not come from
+    // a peerconnection (see LocalStatsCollector.js)
+    if (tpc === null || activeTpc === tpc) {
```

Comment:
I don't understand the comment (LocalStatsCollector.js doesn't call this method directly and I can't follow the entire chain which leads to it).